Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
the right one is "quieres".

btw, now I'm participating in google translator team and checked what is the right translation with others members, and the right translation for "do you want" is "quieres".

you can verify it now in google translator or in bing translator also.